downgrade

```diff
@@ -305,6 +305,7 @@ declare -a to_ignore
 declare -a to_install
 declare -a pacman_args
 declare -a installed
+declare current
```

Comment:
Nice fix. I have a sneaking suspicion that `installed` and `current` could be made local to the `present_packages` function, but it would have to be re-imagined a little bit with something clever like building a `sed` that replaces current and installed package names with their decorated versions, so you could feed `output_package` through it. :thinking:

Reply:
Thank you. Hmm, yes this could be a fix to reduce the number of global variables. We could address this in another PR.